### DISCUSSION TITLE
HEVC can't direct play in most environments, might be good to add to default ReEncodeVideo list

### DISCUSSION THREAD

**Discussion by sam0x17** [ptr727/PlexCleaner#131] [General]
I was processing a large list of files and I noticed some were HEVC and didn't get automatically converted. HEVC as far as I can tell doesn't work in direct play in any players I've tried, so I added it to my list as follows:
```
    "ReEncodeVideo": [
    ...
      {
        "Format": "hevc"
      },
     ...
```

and now I have no issues. Just thought I'd report this as it might be a good idea to add it to the defaults

**Comment by ptr727** [comment]
Interesting, I have lots of hevc / h265 content that directplays on NVidia Shield, I assume you converted it back to h264?

**Comment by sam0x17** [comment]
Correct. Also I didn’t realize HEVC is x265
